### PR TITLE
DOC-790 Enterprise licensing for Redpanda Connect

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves [DOC-790](https://redpandadata.atlassian.net/browse/DOC-790)
Review deadline: December 2nd

## Page previews

[Connector Support Levels](https://deploy-preview-114--redpanda-connect.netlify.app/redpanda-connect/components/connector-support-levels/)
[Licensing](https://deploy-preview-114--redpanda-connect.netlify.app/redpanda-connect/get-started/licensing/)

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-790]: https://redpandadata.atlassian.net/browse/DOC-790?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/get-started/pages/licensing.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+== Apply a license key to Redpanda Connect
+
+Once you have downloaded a license key, you can apply it in the following ways:
+
+. Save the license key to any local directory and run the following command:
+
++
+```bash
+rpk connect run --redpanda-license
```
Comment (Jeffail) [inline modules/get-started/pages/licensing.adoc:18]: the `--redpanda-license` flag needs the actual contents of the license to work, so it'd look something like this:

```
rpk connect run --redpanda-license "eyJ2ZXJzaW9uIjogMCwgIm9yZyI6ICJtZW93IiwgIn..." ./foo.yaml
```

**Review by Jeffail — CHANGES_REQUESTED** [review]

File: modules/get-started/pages/licensing.adoc (hunk 1 above)
Comment (asimms41) [reply modules/get-started/pages/licensing.adoc]: Updated. But when I tested and added a valid license key, I couldn't see any message about the expiry date.

File: modules/get-started/pages/licensing.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+== Purchase an Enterprise Edition license
+
+You can purchase and download a license from the http://redpanda.com/try-enterprise[Redpanda website^].
+
+== Apply a license key to Redpanda Connect
+
+Once you have downloaded a license key, you can apply it in the following ways:
+
+. Save the license key to any local directory and run the following command:
```
Comment (Jeffail) [inline modules/get-started/pages/licensing.adoc:14]:
```suggestion
. Save the license key to any local directory and run the following command:
```

This line isn't needed for the `--redpanda-license` flag.

Comment (asimms41) [reply modules/get-started/pages/licensing.adoc]: Updated

**Review by Jeffail — APPROVED** [review]

**Review by Feediver1 — APPROVED** [review]

File: modules/components/pages/connector-support-levels.adoc [hunk 3]
```diff
@@ -6,4 +6,4 @@ Each connector has a corresponding level of support, which determines whether it
 - *Certified*: Production-ready and available to purchase with Redpanda support.
 - *Community*: Not considered production-ready and available only for community support in https://redpanda.com/slack[Slack^].
 
-Anyone can use the certified and community connectors. However, Enterprise connectors require an https://redpanda.com/compare-platform-editions[Enterprise Edition license^]. You can evaluate enterprise features with a free 30-day trial. Contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda Sales^] to request a trial license, to extend your trial period, or to purchase an Enterprise Edition license.
\ No newline at end of file
+Anyone can use the certified and community connectors. However, to use Enterprise connectors, you need an xref:get-started:licensing.adoc[Enterprise Edition license].
```
[the remark this answers is not in the document]
Comment (asimms41) [reply modules/components/pages/connector-support-levels.adoc:9]: Updated.

File: modules/get-started/pages/licensing.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
+= Enterprise Licensing 
+:description: Learn how to purchase and apply an Enterprise Edition license.
+
+Anyone can use the certified and community connectors, but you need an Enterprise Edition license to use Enterprise connectors.
```
[the remark this answers is not in the document]
Comment (asimms41) [reply modules/get-started/pages/licensing.adoc:4]: Updated.

**Review by Jeffail — APPROVED** [review]

**Pull request merged**